modules/ROOT/nav.adoc

```diff
@@ -15,7 +15,7 @@
   *** xref:custom-application-alerts.adoc[Custom Application Alerts]
   *** xref:autoscaling-in-cloudhub.adoc[Autoscaling in CloudHub]
  ** xref:managing-applications-on-your-own-servers.adoc[Manage Applications on Your Servers]
-  *** xref:hybrid-schedule-mgmt.adoc[Manage Schedules]
```

Comment:
Why not renaming the file too?

Reply:
@fpiacentini08 - We avoid renaming files because there might be links to it. Also, it sounded like the original reporter thought it should be "Schedules" to match CloudHub. If you think it's important, I can make this change.